pipeline.yml

```diff
@@ -22,8 +22,9 @@ build:
   -
     customized_builder_image:
       registry: artifactory
-      url: artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariejs:dotnetcore2
+      url: artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariejs:dotnetcore3
```

Comment:
has this docker file/docs added to the dynamo build resources repo yet?

Reply:
Yes, feel free to check the last PR that was merged today

Reply:
thanks!